addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -133,7 +133,7 @@ options.registry.ImageOptimize.include({
         // feature.
         const imgShapeContainerEl = this.el.querySelector('.o_we_image_shape');
         if (imgShapeContainerEl) {
-            imgShapeContainerEl.classList.toggle('d-none', !odoo.debug);
+            imgShapeContainerEl.classList.add('d-none');
```

Comment:
Let us just keep this line and explain in the commit message that this feature should always be disabled, even when in debug mode, as it is simply not supported (as the comment explains a few lines above).